### PR TITLE
added videos for ACL 19 (and ACL-presented TACL) closes #735

### LINKED ISSUE #735: ACL 2019 videos are available
They can be [browsed here](vimeo.com/user/46432367/folder/1250297).

They video company provided a CSV file linking Anthology IDs to Vimeo URLs. Does anyone want to do the work of ingesting this? [02 ACL2019.xlsx](https://github.com/acl-org/acl-anthology/files/4098244/02.ACL2019.xlsx)

I also have a list of plenary talks and videos that are not currently linked from the Anthology, but should be (#298): [01 ACL 2019 - All Speeches.xlsx](https://github.com/acl-org/acl-anthology/files/4098219/01.ACL.2019.-.All.Speeches.xlsx)

### PR DESCRIPTION
**Pull request by mjpost**
I went ahead and did #735. I didn't link to plenaries etc since we don't yet have a mechanism for that.

### CODE REVIEW

**Comment by mjpost** [comment]
I added a "permission" attribute to video tags. This is because the videos are always recorded and uploaded. On Vimeo, videos for which permission to distribute was not granted are marked private. We could just choose not to link to them, but I think it's better to explicitly record that permission was not granted, so that someone doesn't later wonder why a number of videos are private and unlinked and then add them.

CC: @mbollmann @akoehn

File: data/xml/schema.rnc [hunk 1]
```diff
@@ -43,7 +43,8 @@ Paper = element paper {
    & element url { xsd:anyURI }?
    & element video {
        attribute href { xsd:anyURI },
-       attribute tag { text }
+       attribute tag { text },
+       attribute permission { text }?
```
Comment (mbollmann) [inline data/xml/schema.rnc]: Unless you envision more complicated cases to arise in the future, I would suggest making this `xsd:boolean` and changing the yes/no to true/false, respectively.

**Comment by mjpost** [comment]
This is ready I believe if someone wants to review.

**Comment by mjpost** [comment]
ping

(I spot-checked a number of them and they look correct. Maybe @akoehn was waiting to find time to do something more thorough?)

**Comment by akoehn** [comment]
Additional approval by Marcel not needed after all.  I am not merging it so you can decide whether to rebase, merge or squash, @mjpost.

**Comment by mjpost** [comment]
Thanks, all!

**Pull request merged**